assets/src/edit-story/components/header/titleMissingDialog.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Plain } from '../button';
+import Dialog from '../dialog';
+import Link from '../link';
+
+const Paragraph = styled.p`
+  font-family: ${({ theme }) => theme.fonts.body1.family};
+  font-size: ${({ theme }) => theme.fonts.body1.size};
+  line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.body1.letterSpacing};
+`;
+
+function TitleMissingDialog({ open, onIgnore, onFix, onClose }) {
+  const link = __(
+    'https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/#title',
```

Comment:
Should we try and make this translatable? There different versions of the page for different languages. 

See the docs site.

<img width="157" alt="Screenshot 2020-09-08 at 10 55 26" src="https://user-images.githubusercontent.com/237508/92462009-d3a4da80-f1c1-11ea-8fb3-e40b1e8e8ede.png">

Reply:
@spacedmonkey It's already translatable. See the `__()` above.

Reply:
Can urls be translated like this? If so, I didn't know. Should this has a translation note to it is clear what this link is?

Reply:
Yes they can. And no, a translators comment is not needed here.